### DISCUSSION TITLE
Issues with OR Clauses

### DISCUSSION THREAD

**Discussion by melissajhansen** [jamessimone/apex-rollup#253] [Q&A] [answered]
Are there any limitations or restrictions with using OR clauses in the "Calc Item Where Clause"?  I have some rollups that I converted from DLRS using the script, and I'm getting this error: unexpected token: OR

I added a debug statement and I don't see anything obviously wrong with the query, am I missing something? Debug log attached.

[apex-07L8B000001h325UAA.log](https://github.com/jamessimone/apex-rollup/files/8137199/apex-07L8B000001h325UAA.log)

**Comment by jamessimone** [comment, marked as answer]
@melissajhansen interesting, this is one I haven't seen before. It should be a pretty easy fix.

**Reply by jamessimone** [reply]
I created #254 to fix this!

**Reply by jamessimone** [reply]
@melissajhansen this issue should be resolved in the latest release, #255

**Reply by melissajhansen** [reply]
@jamessimone awesome!! Thanks James!
